api/envoy/api/v2/core/base.proto

```diff
@@ -181,9 +182,16 @@ message TransportSocket {
   // socket implementation.
   string name = 1 [(validate.rules).string.min_bytes = 1];
 
-  // Implementation specific configuration which depends on the implementation being instantiated.
-  // See the supported transport socket implementations for further documentation.
-  google.protobuf.Struct config = 2;
+  oneof config_type {
+    // Implementation specific configuration which depends on the implementation being instantiated.
```

Comment:
Should we associate the comments with the `oneof`? Might need to fix `protodoc` for this if it can't do this factor distribution.

Reply:
Yeah protodoc will swallow those comment with oneof itself.